Azure/communication-ui-library-ios
# [Refactor] Add Test Cases for Camera Pause

## Purpose

For issues we fixed in https://github.com/Azure/communication-ui-library-ios/pull/348, we need an unit test for it as well.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
1. run the test case to make sure it passes
2. change dispatch action to something else and make sure the test does fail 

## Other Information
N/A
